### DISCUSSION TITLE
Raspberry Pi (arm) file extension missing direct install

### DISCUSSION THREAD

**Discussion by 2024OCT13** [stashapp/stash#5374] [Q&A] [answered]
Total noob here, I am lost in what will likely be a simple error to any informed user.  I searched the forums and could not find a similar issue likely all others smarter than I, so any assistance is appreciated.  my challenge is I was trying to install on my raspberry pi (I prefer not to use docker) I followed the instructions to the best of my ability, but it will not install even after chmod, error reads command not found.  I type in just the name of the file, I am newer to linux more familiar with windows and an observation I had is windows files have either a .exe or .msi file extension for running/installing and when I look at this file there is no extension (.bin?), so is it compressed (tar), is there another command I should be using, or should I rename the file with an extension and if so what extension?
![image](https://github.com/user-attachments/assets/020f0363-f37d-42fe-8fa4-3f9cc3509f44)

**Comment by feederbox826** [comment, marked as answer]
It's not a dynamic executable, it should be statically linked, meaning you likely are just running it incorrectly. Here are the steps I took to get up and running on pi(like)

```sh
wget https://github.com/stashapp/stash/releases/download/v0.27.1/stash-linux-arm32v6
chmod +x stash-linux-arm32v6
./stash-linux-arm32v6
```

maybe you're just missing the `./`?

**Reply by 2024OCT13** [reply]
Thank you, I apologize for my error, I missed that little bit of type and now it works, much appreciated.
